main.py

```diff
@@ -101,6 +104,35 @@ def update_kontaktdaten_interactive(
             input_kontaktdaten_key(
                 kontaktdaten, ["kontakt", "notificationReceiver"], "> Mail: ")
 
+        if "notifications" not in kontaktdaten:
+            kontaktdaten["notifications"] = {}
+        if configure_notifications:
```

Comment:
Da der Key jetzt optional ist, können wir ihn einfach komplett weglassen, wenn `configure_notifications` nicht gesetzt ist.

```suggestion
        if configure_notifications:
            if "notifications" not in kontaktdaten:
                kontaktdaten["notifications"] = {}
```